lib/Controller/AccountApiController.php

```
@@ -54,17 +56,31 @@ public function list(): DataResponse {
-					'email' => $alias->getAlias(),
-					'name' => $alias->getName(),
-				], $aliases),
-			];
-		}, $accounts));
+		$result = array_map(function (Account $account) use ($userId) {
+			return $this->transformAccountList($account, false);
+		}, $accounts);
+
+		$delegatedAccounts = $this->accountService->findDelegatedAccounts($userId);
+		$delegatedList = array_map(function (Account $account) use ($userId) {
+			$ownerUserId = $account->getUserId();
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$ownerUserId'. (undefined)

(UnusedLocalVariable)